.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea
+.gradle
+build
+target
+*.ear
+*.jar
+*.iml
+*.ipr
+*.iws
```

Comment:
If you wanna be really cool: https://www.gitignore.io/api/scala,intellij,java,gradle